src/main/java/org/osiam/resources/exceptions/OsiamExceptionHandler.java

```diff
@@ -36,10 +35,11 @@
 
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 
+
 @ControllerAdvice
 public class OsiamExceptionHandler extends ResponseEntityExceptionHandler {
 
-    private static final Logger LOGGER = Logger.getLogger(OsiamExceptionHandler.class.getName());
+    private static final Logger LOGGER = LoggerFactory.getLogger(OsiamExceptionHandler.class);
```

Comment:
Please use class.getName(), because of consistency reasons (auth-server) and slf4j [use the name anyway](http://grepcode.com/file/repo1.maven.org/maven2/org.slf4j/slf4j-api/1.6.1/org/slf4j/LoggerFactory.java#254)